tensorflow/core/api_def/base_api/api_def_ExtractVolumePatches.pbtxt

```diff
@@ -0,0 +1,49 @@
+op {
+  graph_op_name: "ExtractVolumePatches"
+  in_arg {
+    name: "images"
```

Comment:
`images` makes too strong assumption about the type of the input, just `input` is better, or maybe you have any other name specific for volume/cuboid convolutions?